languagetool-http-client/src/main/java/org/languagetool/remote/SentenceAnnotator.java

```diff
@@ -115,14 +115,11 @@ private static void runAnnotation(AnnotatorConfig cfg) throws IOException, NoSuc
         break;
       }
       String[] partsLine = line.split("\t");
-      String sentence;
-      String sentenceHash = "Undefined";
+      String sentence = partsLine[0];
+      String sentenceID = "N/A";
+      String sentenceHash = md5FromSentence(sentence);
       if (partsLine.length == 2) {
-        sentence = partsLine[1];
-        sentenceHash = partsLine[0];
-      } else {
-        sentence = partsLine[0];
-        //sentenceHash = md5FromSentence(sentence);
+        sentenceID = partsLine[1];
```

Comment:
In the new GEMA files the `SentenceID` is the *first* column, not the second. So it should be the other way around.

Reply:
Again. This is the input. 

I will change the output. 

Is the input correct as it is? Sentence in the first column, ID in the second one.